utils/tests/online.py

```diff
@@ -128,7 +128,7 @@ def test_purge_old(self):
         self.redis_conn.set('default_counter:2015-10', 'stuff2')
 
         tc.purge_old()
-        self.assertEqual(['default_counter:2015-10'],
+        self.assertEqual([b'default_counter:2015-10'],
```

Comment:
This explains whats happening with the tests:
```
Python 3.6.1 (default, Apr  4 2017, 09:40:21)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>
>>> [b'default_counter:2015-10']==['default_counter:2015-10']
False
>>>
>>> [b'default_counter:2015-10']==[b'default_counter:2015-10']
True
>>>
```

```
Python 2.7.13 (default, Apr  4 2017, 08:47:57)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>
>>> [b'default_counter:2015-10']==['default_counter:2015-10']
True
>>>
>>> [b'default_counter:2015-10']==[b'default_counter:2015-10']
True
>>>
```
So changing the test string as I did allows the test to pass in py2 and py3, but obviously that is just patching the difference in the test case and not dealing with it directly.